cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -765,32 +765,88 @@
   </property>
 
   <property>
-    <name>kafka.log.dir</name>
+    <name>kafka.server.log.dir</name>
     <value>/tmp/kafka-logs</value>
     <description>
       CDAP Kafka service log storage directory
     </description>
   </property>
 
   <property>
-    <name>kafka.num.partitions</name>
+    <name>kafka.server.log.flush.interval.messages</name>
+    <value>10000</value>
+    <description>
+      The interval length at which we will force an fsync of data written to the log.
```

Comment:
I suggest (we don't usually use "we"; no trailing period on single-sentence descriptions):
```
The number of messages that will be accumulated in memory before writing them to the log file
```